spring-data-geode-test/src/main/java/org/springframework/data/gemfire/tests/integration/ForkingClientServerIntegrationTestsSupport.java

```diff
@@ -53,70 +51,75 @@
 @SuppressWarnings("unused")
 public abstract class ForkingClientServerIntegrationTestsSupport extends ClientServerIntegrationTestsSupport {
 
-	private static ProcessWrapper gemfireServer;
+    private static List<ProcessWrapper> processes = new LinkedList<>();
```

Comment:
`ArrayList` should suffice and is more efficient than a `LinkedList`.

Reply:
Noted, and I'll change. BUT, to talk about efficiency, LinkedLists for this use case is more "correct". Given that we don't need to have any indexing. In this case all that we need to do, is add a server and then iterate over the list and then stop. From an efficiency perspective, LinkedList would be more efficient.. (unless my knowledge of data structures and java data structures is THAT old)